build.gradle

```diff
@@ -41,13 +41,13 @@ repositories {
 
 dependencies {
   runtime 'args4j:args4j:2.0.26'
-  runtime 'com.google.code.gson:gson:2.3.1'
+  runtime 'com.google.code.gson:gson:2.4.0'
 
-  compile 'com.google.javascript:closure-compiler:v20161024'
-  compile 'com.google.guava:guava:18.0'
+  compile 'com.google.javascript:closure-compiler:v20161201'
```

Comment:
Do we use the same updated version inside g3?  I imagine this .gradle file is ignored for the internal build?